INSTALL.rst

```diff
@@ -0,0 +1,355 @@
+.. The source of this document is INSTALL.rst. During the doc build process,
+.. this file is copied over to doc/users/installing.rst.
+.. Therefore, you must edit INSTALL.rst, *not* doc/users/installing.rst!
+
+.. _pip: https://pypi.python.org/pypi/pip/
+
+==========
+Installing
+==========
+
+.. note::
+
+    If you wish to contribute to the project, it's recommended you
+    :ref:`install the latest development version<install_from_source>`.
+
+
+.. contents..
```

Comment:
should be a colon? http://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#explicit-links
or just drop this target?